File: branches/Andean/Fantasy.Base/Tracking/ITrackManager.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Fantasy.Tracking
{
    public interface ITrackManager
    {

        TrackFactory Connection { get; }

        TrackMetaData[] GetActivedTrackMetaData();

        event EventHandler<TrackActivedEventArgs> TrackActived; 

    }
}

File: branches/Stheno/Fantasy.Base/src/fantasy/tracking/ITrackManager.java
package fantasy.tracking;

public interface ITrackManager
{

	TrackFactory getConnection();

	TrackMetaData[] GetActivedTrackMetaData();

//C# TO JAVA CONVERTER TODO TASK: Events are not available in Java:
//	event EventHandler<TrackActivedEventArgs> TrackActived;

}